pontoon/base/models.py

```diff
@@ -89,17 +90,40 @@ def translations_count(query=None):
         )
 
 
-# User class extensions
+@property
+def user_translated_locales(self):
```

Comment:
Could you use `get_objects_for_user` along with the permissions to build this instead? https://github.com/django-guardian/django-guardian/blob/devel/guardian/shortcuts.py#L287